x-pack/plugins/security_solution/public/resolver/view/panel.test.tsx

```diff
@@ -36,6 +31,14 @@ describe(`Resolver: when analyzing a tree with no ancestors and two children and
 
   beforeEach(() => {
     // create a mock data access layer
+    Object.assign(navigator, {
+      clipboard: {
+        writeText: () => {},
```

Comment:
Instead of doing this, would you add `writeText` to the side effect simulator?

Reply:
done

Reply:
done